src/containers/Dashboard.vue

```diff
@@ -341,6 +355,10 @@ export default defineComponent({
     closeTour() {
       this.showTourEndWindow = false;
     },
+
+    closeGiveawayModal() {
+      this.showGiveawayModal = false;
+    },
```

Comment:
![image](https://github.com/user-attachments/assets/4c0b7919-46ce-4e80-875c-c4e84171dd61)
Tested it out with my own account. After refresh, the modal does not show up.
Also, the modal size is good with different screen sizes aswell.